eggo/fabric_util.py

```diff
@@ -106,6 +105,8 @@ def _install_eggo():
     with cd('eggo'):
         if os.environ.get('EGGO_EXP', 'FALSE') == 'TRUE':
             run('git checkout -b experiment origin/experiment')
+        elif os.environ.get('EGGO_FRANK_EXP', 'FALSE') == 'TRUE':
+            run('git checkout -b 1000g-29-samples origin/1000g-29-samples')
```

Comment:
Yeah, I wasn't loving it. What _could_ work is allowing users to pass flags on the command line to override the fork and branch.